src/backend/data/feed.js

```diff
@@ -24,6 +24,7 @@ class Feed {
     this.id = urlToId(url);
     this.author = author;
     this.url = url;
+    this.link = link === '' ? null : link;
```

Comment:
What's the rationale here for storing the empty string, just to make the frontend easier to work on?  Wouldn't we still need to do `if(feed.link)`?  If so, `null` is fine.